File: Solutions/Java/Check_if_strings_are_rotations_of_each_other_or_not.java
class Solution
{
    //Function to check if two strings are rotations of each other or not.
    public static boolean areRotations(String s1, String s2 )
    {
        // Your code here
        if (s1.length() == s2.length()) {
            String txt = s1 + s1;
            if (txt.contains(s2)) {
                return true;
            }
        }
        return false;
    }
}

File: Solutions/Python/Check_if_strings_are_rotations_of_each_other_or_not.py
class Solution:
    
    #Function to check if two strings are rotations of each other or not.
    def areRotations(self,s1,s2):
        #code here
        if len(s1) == len(s2):
            if s2 in (s1+s1):
                return True
        return False
